File: 1431-kids-with-the-greatest-number-of-candies/1431-kids-with-the-greatest-number-of-candies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        target = max(candies)
        for i in range(len(candies)):
            if candies[i]+extraCandies >= target:
                candies[i] = True
            else:
                candies[i] = False
        return candies

File: 1431-kids-with-the-greatest-number-of-candies/1431-kids-with-the-greatest-number-of-candies.go
func kidsWithCandies(candies []int, extraCandies int) []bool {
    max_can := -999999999
    for _, i := range(candies){
        if i > max_can{
            max_can = i
        }
    }
    res := make([]bool, len(candies))
    var tmp int
    
    for ind,i := range candies{
        tmp = i + extraCandies
        if tmp >= max_can{
            res[ind] = true
        }else{
            res[ind] = false
        }
    }
    return res
}